bin/deploy

```diff
@@ -5,82 +5,98 @@ BIN_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
 
 function usage {
   cat <<EOS
-Usage: ./$SCRIPT_NAME <connection-string> <environment>
- e.g.: ./$SCRIPT_NAME core@api.biketag-staging.jackpine.me staging
+Usage: $SCRIPT_NAME <connection-string> <environment>
+ e.g.: $SCRIPT_NAME core@api.biketag-staging.jackpine.me staging
+
+You can deploy against a local docker-machine, but make sure you have the auth keys
+ e.g.: ssh-add ~/.docker/machine/machines/default/id_rsa
+ e.g.: $SCRIPT_NAME docker@\`docker-machine ip\` development
 EOS
 }
 
 # Must be absolute path
```

Comment:
I guess I do consider "~" an absolute path, the logic being that it doesn't matter what directory you are in when you use it.

I also wasn't aware of the discrepancy between `~` vs `${HOME}` thanks for the heads up. It can become an issue when you are trying to do something like string interpolation.